src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -92,6 +92,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Reference Include="System.Xml"/>
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- Do a double-exclude dance to filter PackageReference items down to just NETStandard.Library if it exists -->
+    <_PackageReferenceExceptNETStandardLibrary
+        Include="@(PackageReference)"
+        Exclude="NETStandard.Library"/>
+    <_NETStandardLibraryPackageReference 
+        Include="@(PackageReference)"
+        Exclude="@(_PackageReferenceExceptNETStandardLibrary)"/>
+  </ItemGroup>
+
+  <!-- Automatically reference NETStandard.Library package if target framework is .NETStandard and the package isn't already referenced -->
+  <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETStandard' and '@(_NETStandardLibraryPackageReference)' == ''">
+    <PackageReference Include="NETStandard.Library" Version="1.6.0"/>
```

Comment:
Hard coding 1.6.0? Don't we want to match the major and minor of the netstandard TFM if it's > 1.6.0?